frontend/src/util/reportTools.ts

```diff
@@ -132,13 +132,15 @@ export function useReportTools() {
         erhebungsstelle: Erhebungsstelle,
         artikel: string,
         type: string,
-        base64: string,
+        base64: string | undefined,
         heading: boolean
     ): void {
         if (heading) {
             addHeadingToReport(erhebungsstelle);
         }
-        addImageToReport(base64, createCaption(erhebungsstelle, type));
+        if (base64) {
+            addImageToReport(base64, createCaption(erhebungsstelle, type));
+        }
```

Comment:
in das if sollte der teil

```
        store.dispatch("snackbar/showToast", {
            snackbarTextPart1: `${artikel} ${type} wurde dem PDF Report hinzugefügt.`,
            level: Levels.SUCCESS,
```

noch rein und ne fehlermeldung kommen, wenn base64 undefined ist.

Reply:
Done